boards/pulga/Kconfig

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "nrf52840dk" if BOARD_NRF52840DK
```

```suggestion
    default "pluga" if BOARD_PLUGA
```

I wonder if `pluga-core-v2` would be a better name though

---

It could be, however currently it works both with the Pulga Core V2.0 and the Pulga Core X V1.0 (which is not on the website yet, although it is the version that is likely to be released as official). So I think we could keep it just `pulga` for now, and then add versions in the future.